src/Encamina.Enmarcha.AspNet.Mvc/Bindings/CustomDateTimeModelBinderProvider.cs

```diff
@@ -8,20 +8,20 @@ namespace Encamina.Enmarcha.AspNet.Mvc.Bindings;
 /// </summary>
 public class CustomDateTimeModelBinderProvider : IModelBinderProvider
 {
-    private readonly string customDateTimeFormat;
+    private readonly string? customDateTimeFormat;
```

Comment:
This change has generated a new warning on line 33 `? new CustomDateTimeModelBinder(customDateTimeFormat)`.

We should also change `CustomDateTimeModelBinder` to accept `null` in the `customDateTimeFormat` parameter (and property)